src/WebJobs.Script/Utility.cs

```diff
@@ -996,6 +997,56 @@ public static FunctionAppContentEditingState GetFunctionAppContentEditingState(I
             }
         }
 
+        public static void GetScaleInstancesToProcess(
+            IEnvironment environment,
+            IFunctionsHostingConfiguration hostingConfiguration,
+            IEnumerable<IScaleMonitor> scaleMonitors,
+            IEnumerable<ITargetScaler> targetScalers,
+            out List<IScaleMonitor> scaleMonitrsToProcess,
+            out List<ITargetScaler> targetScalersToProcess)
+        {
+            scaleMonitrsToProcess = new List<IScaleMonitor>();
+            targetScalersToProcess = new List<ITargetScaler>();
+
+            string value = environment.GetEnvironmentVariableOrDefault(EnvironmentSettingNames.TargetBaseScalingEnabled, "0");
+
+            // Check if TBS enabled on stamp level
+            if (value == "1")
+            {
+                string flags = hostingConfiguration.GetValue(ScriptConstants.ScaleControllerFeatureFlags, null).ToLower();
+                if (!string.IsNullOrEmpty(flags))
+                {
+                    foreach (var scaler in targetScalers)
+                    {
+                        if (flags.Contains(scaler.GetType().Name.ToLower()))
```

Comment:
To ensure we don't have casing issues, recommend doing a case insensitive search. That way we don't have to lowercase the feature flags which makes them less readable

Reply:
The internal dictionary is case insensitive so we do need to lower case.

https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Config/FunctionsHostingConfiguration.cs#L100